article/sections/_7_latent_confounding.tex

```diff
@@ -18,18 +18,20 @@ \section{Latent Confounding}
 \subsection{Latent Variable Models}
 \label{intro-latent-models}
 % TODO: Mechanics of how latent variable models control for unobserved confounding.
-Latent variables have been used time and again in different applications of statistical and econometric models in different fields, ranging from epidemiology to transportation.
-A latent variable is defined as a covariate not directly observed about subjects in the sample.
-Examples of these variables include measurements about happiness, lifestyle, economic expectation, 
-morale, among others in many different fields.
+Latent confounders are a special case of latent variables.
+Latent variables include both unobserved and not inherently unobserved variables.
+Examples of the former include measurements about happiness, lifestyle, economic expectation, morale, among many others.
+Examples of the latter include measurement errors (such as inexact measurements of key variables, data entry errors, or measurement inaccuracy), missing data, survey item non-response, among other possible examples.
 
 Latent variable models are based on the assumption that observed variables are independent conditional on latent variables.
```

Comment:
Hmm, that's not true in the case of ICLV's where the various latent variables are typically thought to be independent conditional on the observed variables.

Reply:
This can be closed now, right?

Reply:
Yep. See changes in #177. Thanks @bouzaghrane.